pages/dao/[symbol]/proposal/new.tsx

```diff
@@ -571,6 +573,20 @@ const New = () => {
             governance={governance}
           ></CreateVsrRegistrar>
         )
+      case Instructions.CreateTokenMetadata:
+        return (
+          <CreateTokenMetadata
+            index={idx}
+            governance={governance}
+          ></CreateTokenMetadata>
+        )
+      case Instructions.UpdateTokenMetadata:
+        return (
+          <UpdateTokenMetadata
+            index={idx}
+            governance={governance}
+          ></UpdateTokenMetadata>
```

Comment:
self-closing tag please

Reply:
Thank you for your review!
I fixed it now.